features/step_definitions/generic_steps.rb

```diff
@@ -2,6 +2,10 @@
   unselect(item, from: selector)
 end
 
+When /^(\d) minutes have passed$/ do |number|
```

Comment:
We have cucumber-timecop, so we can use the step `When it is currently x minutes from now`, without defining this one.